msteams-platform/graph-api/rsc/grant-resource-specific-consent.md

```diff
@@ -351,10 +351,17 @@ The tenant-level controls of application RSC permissions vary based on the resou
 
 For delegated permissions, any authorized user can consent to the permissions requested by the app.
 
-<br>
+>[!WARNING]
+>The way you manage settings for team RSC and chat RSC permissions is changing in 2024. Instructions below have been updated to include dates for when new settings will be available for admins to modify
+
```

Comment:
Need to provide disclaimer somewhere that gov clouds still need to use old settings.